web/main/test/test_export.py

```diff
@@ -124,10 +124,11 @@ def test_export(
                         assert_html_equal(file_data, comparison_data)
 
 
-def test_printable_html_livesetting_required(admin_user_factory, client, casebook_factory):
+def test_printable_html_livesetting_required(
+    admin_user_factory, client, full_casebook_parts_factory
+):
     """The printable HTML view requires auth and an explicit setting at this time"""
-    casebook = casebook_factory()
-
+    casebook = full_casebook_parts_factory()[0]
```

Comment:
If you want, you can use the [`full_casebook`](https://github.com/harvard-lil/h2o/blob/develop/web/conftest.py#L545) fixture instead: I think it's exactly the same as `full_casebook_parts_factory()[0]`